server.py

```diff
@@ -17,33 +30,83 @@ def success(name):
 @app.route('/login', methods=('GET', 'POST'))
 def login():
     if request.method == 'POST':
-        username = request.form['username']
-        passw = request.form['password']
 
-        loginAtempt = accounts.ViewAccountController.login(username, passw)
+        loginAccount = accounts.Account()
+
+        loginAccount.username = request.form['username']
+        loginAccount.password = request.form['password']
+
+        loginAtempt = user_authentication.logging(loginAccount)
 
-        if loginAtempt == True: 
+        if loginAtempt[0] == True: 
 
-            #create new account controller
-            #AcController = accounts.AccountController() 
-            print("Server Sees Logged In")
+
+            #set activeUser's attributes to the database values
+            AcController.activeUser = accounts.AccountController.readAccount(loginAtempt[1])
+
+            print("Account logged in is: {}".format(AcController.activeUser), file=sys.stdout) 
+            print("Account Type is: {}".format(AcController.activeUser.account_type), file=sys.stdout) 
+
+            #testing get all subs
+            # allSubs = AcController.getSubscriptions()
+
+            # print("All Subs are: {}".format(*allSubs), file=sys.stdout)
+
+            #end testing
+
+            print("Server Sees Logged In. accountID is: {}".format(loginAtempt[1]), file=sys.stdout)
             return render_template('logged_in.html')
-            #AcController.activeUser.accountID = loginAtempt[2]
-            
+
         else: 
-            return redirect(url_for('LoginFailed', name=user))
+  
+            print("DB Error: {}", loginAtempt[1], file=sys.stdout) #Debugging: sql error to terminal
```

Comment:
error type is not only about database

Reply:
The output from that statement is the exception returned from SQL when checking the login credentials in the database, so yes it is always a database error. Besides, that statement is only output to the terminal, not the application.